holoviews/plotting/bokeh/widgets.py

```diff
@@ -36,22 +36,16 @@ def _plot_figure(self, idx, fig_format='json'):
         """
         self.plot.update(idx)
         if self.embed or fig_format == 'html':
-            html = self.renderer.html(self.plot, fig_format)
-            return html
-        else:
-            doc = self.plot.document
-            if hasattr(doc, 'last_comms_handle'):
-                handle = doc.last_comms_handle
+            if fig_format == 'html':
+                msg = self.renderer.html(self.plot, fig_format)
             else:
-                handle = _CommsHandle(get_comms(doc.last_comms_target),
-                                      doc, doc.to_json())
-                doc.last_comms_handle = handle
-
-            plotobjects = [h for handles in self.plot.traverse(lambda x: x.current_handles)
-                           for h in handles]
-            msg = compute_static_patch(doc, plotobjects)
-            handle.comms.send(json.dumps(msg))
-            return 'Complete'
```

Comment:
I would consider a better name than 'Complete'. Maybe 'Transferred', 'Transfer Complete' or maybe just 'Sent'. Not too important though...